sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1915,11 +1873,15 @@ export interface ContainerProperties {
     hasLegalHold?: boolean;
     // (undocumented)
     lastModified: Date;
+    // (undocumented)
     leaseDuration?: LeaseDurationType;
+    // (undocumented)
     leaseState?: LeaseStateType;
+    // (undocumented)
     leaseStatus?: LeaseStatusType;
     // (undocumented)
     preventEncryptionScopeOverride?: boolean;
+    // (undocumented)
```

Comment:
Please figure out why the comments get lost.

Reply:
The previous comments are added by the V5 generator, there's no description in Swagger. and most of them doesn't mean anything. 
For example: Comments for BlobFlatListSegment is ` An interface representing BlobFlatListSegment.`, maybe leave it as undocumented makes more sense.

Comments for enumeration type does provides available values (also added by V5 generator), and should be kept. I should add description for these types.